django/db/models/base.py

```diff
@@ -298,6 +298,12 @@ def __new__(cls, name, bases, attrs):
                 else:
                     new_class.add_to_class(field.name, copy.deepcopy(field))
 
+        # Set name for _meta.indexes. We cannot do this in Options.contribute_to_class
```

Comment:
I would like to stick to "till then" than "yet" since the fields have been initialized now at this point. What I want to convey is that fields haven't been added to the model when `contribute_to_class` is called.